core/domain/exp_services.py

```diff
@@ -688,24 +687,6 @@ def apply_change_list(exploration_id, change_list):
                         change.property_name ==
                         exp_domain.STATE_PROPERTY_INTERACTION_SOLUTION):
                     state.update_interaction_solution(change.new_value)
-            elif change.cmd == exp_domain.CMD_ADD_GADGET:
-                exploration.add_gadget(change.gadget_dict, change.panel)
-            elif change.cmd == exp_domain.CMD_RENAME_GADGET:
-                exploration.rename_gadget(
-                    change.old_gadget_name, change.new_gadget_name)
-            elif change.cmd == exp_domain.CMD_DELETE_GADGET:
-                exploration.delete_gadget(change.gadget_name)
-            elif change.cmd == exp_domain.CMD_EDIT_GADGET_PROPERTY:
-                gadget_instance = exploration.get_gadget_instance_by_name(
-                    change.gadget_name)
-                if (change.property_name ==
-                        exp_domain.GADGET_PROPERTY_VISIBILITY):
```

Comment:
Remove this constant in exp_domain? (and GADGET_PROPERTY_CUST_ARGS)

Reply:
Thanks @tjiang11! I thought I put it under the "these four properties are deprecated" part, though -- is that the one you mean? The idea is to prevent others using them as keys in the future.

Reply:
Also, @tjiang11 -- if this sounds OK to you, please go ahead and merge. Don't want to delay it too much since this PR is large and prone to merge conflicts.

Thanks!

Reply:
Ohh I see, okay that makes sense